<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1584</width>
    <height>1448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>1191</width>
      <height>941</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>1161</width>
        <height>591</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QTableView" name="tableView_eleve">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>20</y>
          <width>771</width>
          <height>291</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>PreNom</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>150</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Classe</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>310</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>190</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Sexe</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>270</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>NumTel</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>230</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Ele_nom">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>80</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Ele_prenom">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>110</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Ele_class">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Ele_sexe">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>190</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Ele_adress">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>220</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Ele_numTel">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>270</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Ele_email">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>310</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajout_el">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>400</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_det_el">
        <property name="geometry">
         <rect>
          <x>950</x>
          <y>340</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Details</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_del_el">
        <property name="geometry">
         <rect>
          <x>850</x>
          <y>340</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_mod_eleve">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>400</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>320</y>
          <width>161</width>
          <height>201</height>
         </rect>
        </property>
        <property name="title">
         <string>TRI</string>
        </property>
        <widget class="QPushButton" name="pushButton_eleve_triID">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>40</y>
           <width>83</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_eleve_tri_class">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>90</y>
           <width>83</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>classe</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_eleve_tri_sexe">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>140</y>
           <width>83</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>sexe</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>320</y>
          <width>241</width>
          <height>201</height>
         </rect>
        </property>
        <property name="title">
         <string>recherche</string>
        </property>
        <widget class="QLineEdit" name="lineEdit_eleve_sea_id">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>80</y>
           <width>113</width>
           <height>28</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ID/nom/classe</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_eleve_pdf">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>390</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>GESTION ELEVES</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>460</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Stat</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>350</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_ele_age">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>350</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QCustomPlot" name="customplot" native="true">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>110</y>
          <width>841</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>1071</width>
        <height>691</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QTableView" name="tableView_act">
        <property name="geometry">
         <rect>
          <x>385</x>
          <y>40</y>
          <width>651</width>
          <height>351</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>nom Activite</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>220</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>270</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Etat</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>320</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Adresse</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_act_nom">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>130</y>
          <width>181</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_act_desc">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>170</y>
          <width>181</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_act_type">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>210</y>
          <width>181</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_act_etat">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>260</y>
          <width>181</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_act_adr">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>310</y>
          <width>181</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajout_act">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>550</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_del_act">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>430</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_mod_act">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>550</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>420</y>
          <width>241</width>
          <height>201</height>
         </rect>
        </property>
        <property name="title">
         <string>recherche</string>
        </property>
        <widget class="QLineEdit" name="lineEdit_act_sea_id">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>90</y>
           <width>113</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>ID/nom/type</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>420</y>
          <width>161</width>
          <height>201</height>
         </rect>
        </property>
        <property name="title">
         <string>TRI</string>
        </property>
        <widget class="QPushButton" name="pushButton_act_triID">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>40</y>
           <width>83</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_act_trinom">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>90</y>
           <width>83</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>NOM</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_act_tri_nbrP">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>140</y>
           <width>83</width>
           <height>29</height>
          </rect>
         </property>
         <property name="text">
          <string>nbrP</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton_act_pdf">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>490</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>GESTION ACTIVITES</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_stat_act">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>550</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>stat</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>1001</width>
        <height>501</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>120</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ID Eleve</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>120</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>ID Activite</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_id_eleve">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>170</y>
          <width>82</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_id_act">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>170</y>
          <width>82</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_particp">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>20</y>
          <width>451</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajout_part">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>300</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_det_part">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>330</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Detail</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Del_part">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>330</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_mod_part">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>300</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_tri_part_act">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>340</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>tri id activite</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_tri_part_eleve">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>390</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>tri id eleve</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_to_eleve">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>50</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Eleves</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_to_act">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>50</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Activites</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_to_part">
    <property name="geometry">
     <rect>
      <x>1382</x>
      <y>90</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Participation</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1584</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
